src/abv/inventory_api/style_cache.py

```diff
@@ -0,0 +1,30 @@
+
+"""This class creates a style cache that holds recently searched for beers."""
+class StyleCache:
+
+    """Cache constructor."""
+    def __init__(self, filename):
+        self.filename = filename
+        self.cache_dict = {}
+
+        with open(self.filename, 'r') as infile:
+            i = 0
+            for line in infile:

```

Comment:
There's probably a better way to skip the first line in the for loop, don't know whether it's worth changing at the moment but we should refactor this.

Reply:
Will look into it.